src/cli/new.ts

```diff
@@ -123,15 +124,30 @@ if (!arg?.tip) {
   })
 }
 
-if (
-  (scriptName || command !== name) &&
-  !contents.includes(`Name:`)
-) {
-  contents = `// Name: ${scriptName || name || ""}
+const defaultMetadataMode = await getUserDefaultMetadataMode()
+
+if (defaultMetadataMode === 'comment') {
+  if (
+    (scriptName || command !== name) &&
+    !contents.includes(`Name:`)
+  ) {
+    contents = `// Name: ${scriptName || name || ""}
 ${contents.startsWith("/") ? contents : "\n" + contents}
 `
+  }
+} else if (defaultMetadataMode === 'convention') {
+  // TODO: Differentiate the export based on the JS/TS default setting and use jsdoc annotation
```

Comment:
[ ] Differentiate the export based on the JS/TS default setting and use jsdoc annotation